services/src/main/java/org/keycloak/protocol/oidc/endpoints/AuthorizationEndpoint.java

```diff
@@ -290,7 +290,6 @@ private void updateAuthenticationSession() {
         if (request.getResponseMode() != null) authenticationSession.setClientNote(OIDCLoginProtocol.RESPONSE_MODE_PARAM, request.getResponseMode());
         if (request.getClaims()!= null) authenticationSession.setClientNote(OIDCLoginProtocol.CLAIMS_PARAM, request.getClaims());
         if (request.getAcr() != null) authenticationSession.setClientNote(OIDCLoginProtocol.ACR_PARAM, request.getAcr());
-        if (request.getDisplay() != null) authenticationSession.setAuthNote(OAuth2Constants.DISPLAY, request.getDisplay());
```

Comment:
Minor concern: Do we really want to remove the `display` support also from this and `AuthorizationEndpointRequest`, `OAuth2Constants` and related classes? The `display` is standard OIDC parameter ttps://openid.net/specs/openid-connect-core-1_0.html#AuthRequest . And it is possible that people already use it in their authenticator providers for some reason (even though we probably have freedom to remove it as it is not documented anywhere). And we might possibly rething and start to use it again in the future.

Or do you prefer to remove it entirely now and then if we use inthe future, re-add it to the codebase?

Reply:
Added it back in just in case someone uses it, and doesn't hurt to have it there I guess